plugin/adsk/plugin/plugin.cpp

```diff
@@ -208,7 +208,9 @@ MStatus initializePlugin(MObject obj)
     registerCommandCheck<MayaUsd::ADSKMayaUSDImportCommand>(plugin);
     registerCommandCheck<MayaUsd::EditTargetCommand>(plugin);
     registerCommandCheck<MayaUsd::LayerEditorCommand>(plugin);
+#if defined(WANT_QT_BUILD)
     registerCommandCheck<MayaUsd::LayerEditorWindowCommand>(plugin);
+#endif
```

Comment:
When no Qt, don't register the command that is used to open the Layer Editor (since we didn't build the layer editor). Note: I left the two commands above in since they are built even when no Qt (they are in a different folder from the "ui/layerEditor"